lib/fuse_lowlevel.c

```diff
@@ -1835,6 +1835,12 @@ static void do_copy_file_range(fuse_req_t req, fuse_ino_t nodeid_in, const void
 		fuse_reply_err(req, ENOSYS);
 }
 
+#define FUSE_MAX_MAX_PAGES 256
+#define FUSE_DEFAULT_MAX_PAGES_PER_REQ 32
+
+/* room needed in buffer to accommodate header */
+#define HEADER_SIZE 0x1000
+
```

Comment:
Please put the #ifdefs into `lib/fuse_i.h`, and use a more expressive name for `HEADER_SIZE`

Reply:
I only moved this up in this file to use it at this point. But I moved it to fuse_i.h now anyway.